src/components/input/Input.jsx

```diff
@@ -0,0 +1,33 @@
+import styles from "./Input.module.css";
+import clsx from "clsx";
+
+const Input = ({
+  placeholder = "placeholder",
+  disabled = false,
+  isError = false,
+  errorMessage = "",
+  value,
+  onChange,
+  ...props
+}) => {
+  return (
+    <div>
+      <input
+        className={clsx(styles.input, {
+          [styles.isError]: isError,
+          [styles.disabled]: disabled,
+        })}
+        value={value}
+        onChange={onChange}
+        placeholder={placeholder}
+        disabled={disabled}
+        {...props}
```

Comment:
1. 사실 상 `input` 이라는 요소의 기본 속성들은 전부 rest 파라미터로 대체 가능하니
props의 가짓수를 더 줄여볼 수 있을 것 같아요. 😅

```suggestion
  isError = false,
  errorMessage = "",
  ...props
}) => {
  return (
    <div>
      <input
        className={clsx(styles.input, {
          [styles.isError]: isError,
        })}
        {...props}
```

2. `disabled` 속성은 `className`과 `input`에 각각 전달할 필요 없이 CSS 가상 선택자로 스타일을 주면 될 것 같아요 😊
```jsx
// Input.jsx

  //  disabled
  isError = false,
  errorMessage = "",
  ...props
}) => {
  return (
    <div>
      <input
        className={clsx(styles.input, {
          [styles.isError]: isError,
          // [styles.diabled]: disabled
        })}
        {...props}
```
```css
/* css */
input:disabled {
  ...
}
```